package.json

```diff
@@ -7,40 +7,40 @@
     "url": "https://github.com/mozilla/chronicle/issues"
   },
   "dependencies": {
-    "bell": "2.0.0",
+    "bell": "2.1.0",
     "boom": "2.6.1",
     "commander": "2.6.0",
-    "convict": "0.6.0",
-    "elasticsearch": "3.1.1",
-    "embedly": "1.0.2",
-    "hapi": "8.1.0",
+    "convict": "0.6.1",
+    "elasticsearch": "3.1.3",
+    "embedly": "1.0.4",
```

Comment:
embedly changes 1.0.2 -> 1.0.4 are kinda funny:

- 1.0.2 -> 1.0.3 rev superagent dependency (which caused a bug that broke the extract API)
- 1.0.3 -> 1.0.4 fix issue caused by bumping dependency, lol https://github.com/embedly/embedly-node/commit/fde857b